sql/step_04_all_following_steps.sql

```diff
@@ -84,3 +84,5 @@ source step_87_alter_table_opd.sql;
 source step_88_alter_table_patient.sql;
 source step_89_alter_tables_innodb.sql;
 source step_90_login_retries.sql;
+source step_91_patient_consensuns_table.sql;
+source step_92_consensus_old_records.sql;
```

Comment:
please merge these two scripts (there's no need to have them separated) as `step_92_patient_consensuns_table.sql` (because there is a new step_91 in core now. Thanks